app/src/main/res/xml/settings.xml

```diff
@@ -68,52 +54,6 @@
             android:summary="@string/preference_performance_block_webfonts_summary"
             android:defaultValue="false" />
 
-        <!-- // Not shipping in v1 (#188)
-        <SwitchPreference
-            android:title="@string/preference_performance_block_images"
-            android:key="@string/pref_key_performance_block_images"
-            android:defaultValue="@string/preference_performance_block_images" />
-        -->
-
-    </PreferenceCategory>
-
-    <PreferenceCategory
-        android:title="@string/preference_category_mozilla">
-
-        <org.mozilla.focus.widget.DefaultBrowserPreference
-            android:layout="@layout/focus_preference"
-            android:key="@string/pref_key_default_browser"
```

Comment:
What about these `@string/pref`'s? the ones ahunt added (for string compare reasons, I don't remember the specifics).

Reply:
They're used in other places for whatever reason – the IDE has removed any that are actually unused.

I was going for "good enough" here rather than trying to get every last piece.